be2-scala/src/test/scala/ch/epfl/pop/pubsub/graph/validator/Lao/CreateLaoDecoderSuite.scala

```diff
@@ -0,0 +1,110 @@
+package ch.epfl.pop.pubsub.graph.validator.lao
+
```

Comment:
I find interesting how the tests look when using the FlatSpec we talked about! Looks good

Reply:
Yes it adds the BDD flavor to it.